macros/get_call_columns.sql

```diff
@@ -26,17 +26,17 @@
     {"name": "updated_at", "datatype": dbt.type_timestamp()}
 ] %}
 
-{% if target.type == 'snowflake' %}
-    {{ columns.append({"name": "FROM", "datatype": dbt.type_string(), "quote": True, "alias": "call_from"}) }}
-{% else %}
-    {{ columns.append({"name": "from", "datatype": dbt.type_string(), "quote": True, "alias": "call_from"}) }}
-{% endif %},
-
-{% if target.type == 'snowflake' %}
-    {{ columns.append({"name": "TO", "datatype": dbt.type_string(), "quote": True, "alias": "call_to"}) }}
-{% else %}
-    {{ columns.append({"name": "to", "datatype": dbt.type_string(), "quote": True, "alias": "call_to"}) }}
-{% endif %},
+{% if target.type == 'snowflake' %} 
+    {{ columns.append({"name": "FROM", "datatype": dbt.type_string(), "quote": True, "alias": "call_from"}) }} 
+{% else %} 
+    {{ columns.append({"name": "from", "datatype": dbt.type_string(), "quote": True, "alias": "call_from"}) }} 
+{% endif %}
+ 
+{% if target.type == 'snowflake' %} 
+    {{ columns.append({"name": "TO", "datatype": dbt.type_string(), "quote": True, "alias": "call_to"}) }} 
+{% else %} 
+    {{ columns.append({"name": "to", "datatype": dbt.type_string(), "quote": True, "alias": "call_to"}) }} 
+{% endif %}
```

Comment:
Is there any reason these are in two separate conditionals? I imagine we could make this dryer by combining them.

Reply:
good point- updated

Reply:
Thanks for applying this update. However, can we make sure the trailing commas are appropriate. It looks like the conditional blocks don't have the commas. It should look like the following.

```sql
    {"name": "trunk_id", "datatype": dbt.type_string()},
    {"name": "updated_at", "datatype": dbt.type_timestamp()},
] %}

{% if target.type == 'snowflake' %}
    {{ columns.append({"name": "FROM", "datatype": dbt.type_string(), "quote": True, "alias": "call_from"}) }},
    {{ columns.append({"name": "TO", "datatype": dbt.type_string(), "quote": True, "alias": "call_to"}) }}
    {% else %}
    {{ columns.append({"name": "from", "datatype": dbt.type_string(), "quote": True, "alias": "call_from"}) }},
    {{ columns.append({"name": "to", "datatype": dbt.type_string(), "quote": True, "alias": "call_to"}) }}
{% endif %}
```

Reply:
Yeah I thought about that! But I noticed in our other packages (ex Amplitude, Jira, Stripe) [Jira ex](https://github.com/fivetran/dbt_jira_source/blob/main/macros/get_issue_field_history_columns.sql), we don't do that. I'll make the change since it doesn't hurt, but we should keep it mind for our other packages..